docs/reference/module-options.md

```diff
@@ -372,3 +372,442 @@ module.exports = {
 }
 ```
 
+## Options for the core page module
+
+Option settings in this section apply to the core page module (`@apostrophecms/page`).
+
+| Option | Value type | Description |
+|---------|---------|---------|
+| [`builders`](#builders) | Object | Set query builder values to be used when pages are served. |
+| [`home`](#home) | Boolean/Object | Change how the home page is added to `req.data` when pages are served. |
+| [`minimumPark`](#minimumpark) | Array | Override home page default properties when created. |
+| [`park`](#park) | Array | Set pages to be created on site start with configuration. |
```

Comment:
"with additional configuration"? I don't know about that for this short description. Additional to what?

Reply:
" Set pages to be created on site **start** with configuration." Would this be site **creation**?

Reply:
@localghost443 It gets semantic, but if a new page was added to this a month after launch, it would be created when the site was restarted, even though the site already was created.

Reply:
Additional to the home page and trash page, but I think we could live without it.